src/components/profile/profile.pw.tsx

```diff
@@ -0,0 +1,187 @@
+  namePreview,
+  profilePreview,
+  initialPreview,
+} from "../../../playwright/components/profile";
+
+import { portraitImage } from "../../../playwright/components/portrait";
+
+import { PROFILE_SIZES } from "../../../src/components/profile/profile.config";
+
+import {
+  checkAccessibility,
+  containsClass,
+} from "../../../playwright/support/helper";
+import { ProfileProps } from "./profile.component";
+
+const testImage =
+  "https://www.gravatar.com/avatar/05c1c705ee45d7ae88b80b3a8866ddaa?s=24&d=404";
+
+const imageURLs = [
+  "https://avataaars.io/?avatarStyle=Transparent&topType=LongHairStraight&accessoriesType=Blank&hairColor=BrownDark&facialHairType=Blank&clotheType=BlazerShirt&eyeType=Default&eyebrowType=Default&mouthType=Default&skinColor=Light",
+  "https://www.gravatar.com/avatar/05c1c705ee45d7ae88b80b3a8866ddaa?s=24&d=404",
+];
+
+const testData = [CHARACTERS.DIACRITICS, CHARACTERS.SPECIALCHARACTERS];
+
+test.describe("should check Profile component properties", () => {
+  test(`should render profile component with className`, async ({
+    mount,
+    page,
+  }) => {
+    await mount(<ProfileComponent className="profile-classnames" />);
+
+    await containsClass(profilePreview(page), "profile-classnames");
+  });
+
+  testData.forEach((email) => {
+    test(`should check when email prop is passed as ${email}`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<ProfileComponent email={email} src={testImage} />);
+
+      await expect(emailPreview(page)).toHaveText(email);
+    });
+  });
+
+  testData.forEach((name) => {
+    test(`should check when name prop is passed as ${name}`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<ProfileComponent name={name} src={testImage} />);
+
+      await expect(namePreview(page)).toHaveText(name);
+    });
+  });
+
+  ([
+    [PROFILE_SIZES[0], 22],
+    [PROFILE_SIZES[1], 30],
+    [PROFILE_SIZES[2], 38],
+    [PROFILE_SIZES[3], 54],
+    [PROFILE_SIZES[4], 70],
+    [PROFILE_SIZES[5], 102],
+    [PROFILE_SIZES[6], 126],
+  ] as [ProfileProps["size"], number][]).forEach(([size, heightAndWidth]) => {
+    test(`should check when size prop is passed as ${size}`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<ProfileComponent size={size} />);
+
+      await expect(initialPreview(page)).toHaveCSS(
+        "height",
+        `${heightAndWidth}px`
+      );
+      await expect(initialPreview(page)).toHaveCSS(
+        "width",
+        `${heightAndWidth}px`
+      );
+    });
+  });
+
+  [
+    ["Dan Jin", "DJ"],
+    ["Sid Ford", "SF"],
+  ].forEach(([name, passInitials]) => {
+    test(`should check name initials when name prop is passed as ${name}`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<ProfileComponent initials={passInitials} name={name} />);
+
+      await expect(initialPreview(page)).toHaveCSS("height", "38px");
+      await expect(initialPreview(page)).toHaveCSS("width", "38px");
```

Comment:
This test is supposed to check the name and initials of the Profile, but only verifies height and width.

Reply:
Has this been applied?